src/providers/fault-count/cat-c/fault-count.cat-c.ts

```diff
@@ -268,4 +244,28 @@ export class FaultCountCHelper {
     return result;
   }
 
+  public static getVehicleChecksFaultCountCatC = (
```

Comment:
we could do with unit tests for all of these. As we don't seem to have any, all bug fixes should have unit tests to make sure that specific thing can't happen again

Reply:
I noticed there is a spec file related to this one (there should be instead of us relying on the entry point provider to test everything)